CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 ## master / unreleased
+ - [ENHANCEMENT] Time-ovelapping blocks are now allowed. [#370](https://github.com/prometheus/tsdb/pull/370)
+   - [ENHANCEMENT] Added `MergeChunks` function in `chunkenc/xor.go` to merge 2 time-overlapping chunks.
+   - [ENHANCEMENT] Added `MergeOverlappingChunks` function in `chunks/chunks.go` to merge multiple time-overlapping Chunk Metas.
+   - [ENHANCEMENT] Added `MinTime` and `MaxTime` method for `BlockReader`.
```

Comment:
something got messed up with the formatting here. As I suggested I prefer to have these described as sub-changes of the main change described in the same [ENHANCEMENT] block rather than in their own separate [ENHANCEMENT] blocks. This way I would know why were these added and easier to know if can be removed as part of another refactoring.

Reply:
yes that looks good like this.